src/base/static/components/molecules/map-legend-item.js

```diff
@@ -1,23 +1,56 @@
 import React from "react";
 import PropTypes from "prop-types";
-import classNames from "classnames";
+import styled from "react-emotion";
 
-import { Icon, Swatch } from "../atoms/feedback";
-import { PanelLabel } from "../atoms/typography";
+import { Icon } from "../atoms/feedback";
+import { RegularLabel } from "../atoms/typography";
 
 import "./map-legend-item.scss";
 
-const MapLegendItem = props => (
-  <div className="map-legend-item">
-    {props.icon && (
-      <div className="map-legend-item__icon-container">
-        <Icon icon={props.icon} classes="map-legend-item__icon" />
-      </div>
-    )}
-    {props.swatch && <Swatch color={props.swatch} />}
-    <PanelLabel classes="map-legend-item__label">{props.label}</PanelLabel>
-  </div>
-);
+const ColorSwatch = styled("span")(props => ({
+  backgroundColor: props.color,
+  width: 20,
+  height: 20,
+  marginRight: 10,
+}));
+
+// TODO: Abstract these components out when we refactor other panel types.
+const IconContainer = styled("div")(props => ({
+  display: "flex",
+  justifyContent: "center",
+  flex: 1,
+}));
+
+const MapLegendItemContainer = styled("div")(props => ({
+  display: "flex",
+  alignItems: "center",
+  marginBottom: 5,
+  paddingLeft: 15,
+}));
+
+const MapLegendItem = props => {
+  return (
+    <MapLegendItemContainer>
+      {props.icon && (
+        <IconContainer>
+          <Icon icon={props.icon} classes="map-legend-item__icon" />
```

Comment:
`Icon` is used elsewhere; I'll refactor this for CSS-in-JS at a later step.

Reply:
Sounds good. Lmk before you implement this refactor so we can brainstorm a better pattern here. I think there is a better approach here than passing in arbitrary paths to the icon file.